### PR TITLE
Register for demo

### PR DESCRIPTION
**Pull request by dtuite**
Users can now insert their email to get a demo of Backstage.

 - [ ] Update the Zapier email template which is sent to people who sign up.
 - [x] Test the newsletter subscription flow on Netlify.
 - [x] Make sure the form at the bottom of the plugins is for newsletter sign up list. Not "Notify me".
 - [x] Set up a Zap to automatically subscribe certain emails to the newsletter on Revue.
 - [x] Add a newsletter link to the header.

### CODE REVIEW

File: src/components/SitewideFooter.js [hunk 1]
```diff
@@ -32,14 +32,21 @@ const useStyles = createUseStyles((theme) => ({
...
     <footer>
       <div className={classes.inner}>
         <div>© {new Date().getFullYear()} Larder, Inc. All rights reserved.</div>
 
         <nav>
-          <Link to="/terms">Terms of service</Link>
+          <a href={newsletterUrl} target="_blank" rel="noopener noreferrer">
+            Backstage Weekly
+          </a>
+          <Link to="/terms" className={classes.nonFirstLink}>
```
Comment (martina-if) [inline src/components/SitewideFooter.js:47]: out of curiosity what is  `classes.nonFirstLink`?

File: src/components/SitewideHeader/HamburgerMenu.js [hunk 2]
```diff
@@ -96,9 +96,12 @@ const useStyles = createUseStyles(() => ({
   },
 }));
 
-const HamburgerMenu = () => {
+const HamburgerMenu = ({ siteMetadata }) => {
   const classes = useStyles();
 
+  const TWITTER_URL = `https://twitter.com/${siteMetadata.social.twitter}`;
```
Comment (martina-if) [inline src/components/SitewideHeader/HamburgerMenu.js:102]: Why not store the whole url in the siteMetadata instead?

**Comment by martina-if** [comment]
The new emoji is also missing for me:

demo-emoji-missing

If I sign up for the demo, I'm not supposed to sign up for the newsletter? Or maybe I had already done that before :/
 
already

**Review by martina-if — DISMISSED** [review]

**Comment by dtuite** [comment]
@martina-if 

Signing up for the demo doesn't automatically sign people up for the newsletter. Doing so would be a GDPR violation unless we explicitly state that the user is going to be signed up to a newsletter. We could add a checkbox under the form which says "sign me up for the newsletter" but that feels like too much clutter for a landing page. I have been manually checking with people if they would like to sign up for the newsletter. It's a manageable solution for now.

I don't understand the issue with the emojis. You use Linux right? What browser are you using? Are they also missing on the production deployment of roadie.io?

File: src/components/SitewideFooter.js (hunk 1 above)
Comment (dtuite) [reply src/components/SitewideFooter.js:47]: I'm adding spacing between the links to keep them apart from each other but the spacing is not required on the first link in the list.

File: src/components/SitewideHeader/HamburgerMenu.js (hunk 2 above)
Comment (dtuite) [reply src/components/SitewideHeader/HamburgerMenu.js:102]: This setup (twitter username but not URL in config) comes with Gatsby by default so I kinda just stuck with it. I guess one advantage is that we have the twitter username available if we ever need that in non-linked form. We can count on the Twitter hostname not changing anytime soon so it's basically static config.

With that said, I'm not actually using the metadata version in the code where I reference the username. I will fix that.

**Comment by martina-if** [comment]
Ok, I think I got that message "The email address has already been subscribed" because I got redirected to the production page instead of the netlify preview. I thought I couldn't sign up for both independently but it's not the case :+1: all good.

**Review by martina-if — APPROVED** [review]

**Pull request merged**